scripts/ci/serve-browser-fixtures.py

```
@@ -0,0 +1,75 @@
+                return
+
+            body = fixture_path.read_bytes()
+            self.send_response(200)
+            self.send_header("Content-Type", "text/html; charset=utf-8")
+            if fixture_name == strict_csp_fixture:
+                self.send_header("Content-Security-Policy", STRICT_CSP)
+            self.send_header("Content-Length", str(len(body)))
+            self.end_headers()
+            self.wfile.write(body)
+
+        def log_message(self, format: str, *args: object) -> None:
```

Ruff (0.16.1)

[error] Function argument `format` is shadowing a Python builtin

(A002)